wallet-core/jacoco.gradle

```diff
@@ -65,10 +65,10 @@ def createVariantCoverage(variant) {
         group = "Reporting"
         description = "Generate Jacoco coverage reports for the ${variantName.capitalize()} build."
 
-        reports {
-            xml.enabled = true
-            html.enabled = true
-        }
+//        reports {
```

Comment:
Do we still need those disabled? In that case I can take a look into this briefly.